src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/caseload/CaCaseloadService.kt

```diff
@@ -193,15 +194,15 @@ class CaCaseloadService(
 
     // If COM username but no code, do a separate call to use the data in CVL if it exists. Should help highlight any desync between Delius and CVL
     val comUsernames = cases.withStaffUsername.map { c -> c.probationPractitioner?.staffUsername!! }
-    val coms = communityApiClient.getStaffDetailsByUsername(comUsernames).associateBy { it.username?.lowercase() }
+    val coms = deliusApiClient.getStaffDetailsByUsername(comUsernames).associateBy { it.username?.lowercase() }
 
     val caCaseListWithStaffUsername = cases.withStaffUsername.map { caCase ->
       val com = coms[caCase.probationPractitioner?.staffUsername?.lowercase()]
       if (com != null) {
         caCase.copy(
           probationPractitioner = ProbationPractitioner(
-            staffCode = com.staffCode,
-            name = com.staff?.let { "${it.forenames} ${it.surname}" },
+            staffCode = com.code,
+            name = com.name?.let { "${it.forename} ${it.surname}" },
```

Comment:
is there a reason this doesn't use `fullName`?

Reply:
Great spot! Ive changed this. Also fixed detekt